Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -222,8 +240,16 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
 
   // encoding settings
   response.preset += `,-map 0 -dn -c:v libx265 -preset ${ffmpegPreset}`
-  + ` -x265-params crf=${crf}:bframes=${bframe}:rc-lookahead=32:ref=6:b-intra=1:aq-mode=3`
-  + ' -a53cc 0 -c:a copy -c:s copy -max_muxing_queue_size 9999';
+  response.preset += ` -x265-params crf=${crf}:bframes=${bframe}:rc-lookahead=32:ref=6:b-intra=1:aq-mode=3`
+  response.preset += ' -a53cc 0 -c:a copy -c:s copy';
```

Comment:
Why did you split this instead of just concating to the original string?

Reply:
I'm not sure of the proper syntax here for multiline. This seemed safer/ more clear.

Reply:
> I'm not sure of the proper syntax here for multiline. This seemed safer/ more clear.

The way it was before was fine.